engine/src/main/scala/cromwell/webservice/MetadataBuilderActor.scala

```diff
@@ -128,41 +128,60 @@ object MetadataBuilderActor {
       case s => Option(s.toInt)
     }
   }
+
+  private case object IndexedEmptyJson extends IndexedJsonValue {
+    override def index: Option[Int] = None
+    override def withIndex(index: String): IndexedJsonValue = this
+    override def toJson: JsValue = throw new IllegalStateException("IndexedEmptyJson cannot be converted to Json")
+  }
```

Comment:
We can actually have a small tech talk about it, but I'm not sure the zero here ought to have the exact same behaviour a zero has in a Monoid.
There also might be something missing/needing to be clarified in the mini language to push an empty list.